packages/commands-http/src/request.ts

```diff
@@ -0,0 +1,84 @@
+import { Command } from '@seagull/commands'
+import fetch, { Response as FetchResponse } from 'node-fetch'
+import { Http, RequestConfig } from './index'
+import SeedStorage from './seedStorage'
+
+/**
+ * Http request command.
+ */
+export class Request<T> extends Command<T> {
+  executeConnected = this.executeCloud
+  executeEdge = this.executeCloud
+
+  private seed: SeedStorage<T>
+
+  /**
+   * Creates an http request command.
+   * @param config Http request command configuration.
+   */
+  constructor(public config: RequestConfig) {
+    super()
+    if (!config) {
```

Comment:
Do we need this check? Because of the type system it is not possible to have no config.

Reply:
The config might be undefined on runtime though. The input checks are just to help finding error sources within logs etc. and seems more solid to me. Any other opinions on this?